docs/dev/development.md

```diff
@@ -314,110 +348,134 @@ def some_method(a: int, b: str) -> float:
+Each job might need different set of requirements and it would be inefficient to
+install a full-blown dev env for every tiny job (e.g. `mypy` check). Instead, in
+the directory `dev_tools/requirements`, create a separate `<job>.env.txt` and
+include the necessary tools in there. Requirements files can include each other,
+which is heavily leveraged in our requirements files in order to remove
+duplication.
 
 You can call the following utility to unroll the content of a file:
 
-```
+```shell
 python dev_tools/requirements/reqs.py dev_tools/requirements/dev.env.txt
 ```
 
 ## Producing a PyPI package
 
-1. Do a dry run with test PyPI.
+1.  Do a dry run with test PyPI.
 
     If you're making a release, you should have access to a test PyPI account
-    capable of uploading packages to Cirq. Put its credentials into the environment
-    variables `TEST_TWINE_USERNAME` and `TEST_TWINE_PASSWORD` then run
+    capable of uploading packages to Cirq. Put its credentials into the
+    environment variables `TEST_TWINE_USERNAME` and `TEST_TWINE_PASSWORD` then
+    run
 
-    ```bash
+    ```shell
     ./dev_tools/packaging/publish-dev-package.sh EXPECTED_VERSION --test
     ```
 
-    You must specify the EXPECTED_VERSION argument to match the version in [cirq/_version.py](https://github.com/quantumlib/Cirq/blob/main/cirq-core/cirq/_version.py), and it must contain the string `dev`.
-    This is to prevent accidentally uploading the wrong version.
+    You must specify the EXPECTED_VERSION argument to match the version in
+    [cirq/\_version.py](https://github.com/quantumlib/Cirq/blob/main/cirq-core/cirq/_version.py),
+    and it must contain the string `dev`. This is to prevent accidentally
+    uploading the wrong version.
 
-    The script will append the current date and time to the expected version number before uploading to test PyPI.
-    It will print out the full version that it uploaded.
-    Take not of this value.
+    The script will append the current date and time to the expected version
+    number before uploading to test PyPI. It will print out the full version
+    that it uploaded. Take not of this value.
 
     Once the package has uploaded, verify that it works
 
-    ```bash
+    ```shell
     ./dev_tools/packaging/verify-published-package.sh FULL_VERSION_REPORTED_BY_PUBLISH_SCRIPT --test
-   ```
+    ```
 
-    The script will create fresh virtual environments, install Cirq and its dependencies, check that code importing Cirq executes, and run the tests over the installed code. If everything goes smoothly, the script will finish by printing `VERIFIED`.
+    The script will create fresh virtual environments, install Cirq and its
+    dependencies, check that code importing Cirq executes, and run the tests
+    over the installed code. If everything goes smoothly, the script will finish
+    by printing `VERIFIED`.
 
-2. Do a dry run with prod PyPI
+1.  Do a dry run with prod PyPI
```

Comment:
please leave the numbering in.  It was correct and easier to read in plain text.